docs/integrations/immer.mdx

```diff
@@ -24,17 +24,20 @@ The signature of writeFunction is `(get, set, update: (draft: Draft<Value>) => v
 import { useAtom } from 'jotai'
 import { atomWithImmer } from 'jotai-immer'
 
-const countAtom = atomWithImmer(0)
+const countAtom = atomWithImmer({ value: 0 })
```

Comment:
This doesn't show what we would like to show. Let's keep `atomWithImmer(0)`.

Reply:
Doesn't work: https://codesandbox.io/s/crazy-goldstine-jl5vdm

Reply:
my bad

Reply:
I don't think `0` adequately demonstrates immer. Immer works well with objects but not with primitives. For example, `++draft` does not work as expected.